### DISCUSSION TITLE
请问如何修改或禁用内容选中的提示工具？

### DISCUSSION THREAD

**Discussion by zxyao145** [wangeditor-team/wangEditor#4231] [Q&A] [answered]
请问如何修改或禁用内容选中的提示工具？类似下图中的框选的部分
![image](https://user-images.githubusercontent.com/24973084/169073843-569fb2b1-07f1-468f-bca7-deaf4403ec38.png)

**Comment by wangfupeng1988** [comment, marked as answer]
参考 https://www.wangeditor.com/v5/editor-config.html#hoverbarkeys

**Reply by zxyao145** [reply]
谢谢！
